amundsen_application/static/js/components/BrowsePage/index.tsx

```diff
@@ -8,12 +8,14 @@ import './styles.scss';
 import AppConfig from 'config/config';
 import LoadingSpinner from 'components/common/LoadingSpinner';
 import TagInfo from 'components/Tags/TagInfo';
-import { Tag } from 'components/Tags/types';
+import { TagsInterface } from 'interfaces/Tags';
 
 import { GlobalState } from 'ducks/rootReducer';
 import { getAllTags } from 'ducks/allTags/reducer';
 import { GetAllTagsRequest } from 'ducks/allTags/types';
 
+type Tag = TagsInterface.Tag;
```

Comment:
I like splitting out the common types in to the interfaces folder, but I have some reservations about the namespaces. It seems like we have to import the namespace, then redefine each type for local usage.

It seems like the main benefit from this would be to avoid naming collisions, but I'm not sure how frequent those will occur, especially given that we should be importing from the common interfaces folder.

Reply:
I've moved away from namespaces entirely because even though folks still to use them, looking towards the future ES6 modules are preferred over namespaces. Folks can still use namespaces within modules if they want to quickly deal with name conflicts, but I think it will be best to not structure such a large change around a feature that sounds like the industry is moving away from.